physionet-django/console/views.py

```diff
@@ -779,7 +779,27 @@ def past_credential_applications(request):
 
     """
     s_applications = CredentialApplication.objects.filter(status=2)
-    u_applications = CredentialApplication.objects.filter(status=1)
+    u_applications = CredentialApplication.objects.filter(status=1).order_by('-application_datetime')
+    if request.method == 'POST':
+        if 'remove_credentialing' in request.POST and request.POST['remove_credentialing'].isdigit():
+            credentiallied_id = request.POST['remove_credentialing']
+            C_application = CredentialApplication.objects.filter(id=credentiallied_id)
+            if C_application:
+                C_application = C_application.get()
+                C_application.user.is_credentialed = False
+                C_application.user.credential_datetime = None
+                C_application.user.save()
+                C_application.decision_datetime = None
+                C_application.status = 1
+                C_application.save()
+        elif 'manage_credentialing' in request.POST and request.POST['manage_credentialing'].isdigit():
+            credentiallied_id = request.POST['manage_credentialing']
```

Comment:
- typo in `credentiallied_id` -> `credentialed_id` or maybe just `cid`
- not sure the `C_application` needs capitalisation
- would there be a problem if `C_application.user.save()` was successful but `C_application.save()` failed? If so, the transaction should be atomic (https://docs.djangoproject.com/en/2.2/topics/db/transactions/)